content/docs/tutorials/at-dude/5-reset-atsign/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: codelab
+
+title: "Reset atsign" # Step Name
+description: How to reset any app built on the atPlatform # SEO Description for this step Documentation
+
+draft: true # TODO CHANGE THIS TO FALSE WHEN YOU ARE READY TO PUBLISH THE PAGE
```

Comment:
If this is finished, then set it to: `draft: false`

Reply:
@CurtlyCritchlow